lib/AST/Attr.cpp

```diff
@@ -920,17 +951,24 @@ bool AvailableAttr::isUnconditionallyDeprecated() const {
   llvm_unreachable("Unhandled PlatformAgnosticAvailabilityKind in switch.");
 }
 
+llvm::VersionTuple AvailableAttr::getAvailKindMinVersion(const ASTContext &ctx) const {
+  if (isLanguageVersionSpecific()) {
+      return ctx.LangOpts.EffectiveLanguageVersion;
+  } else if (isSwiftPMManifestVersionSpecific()) {
+      return ctx.LangOpts.SwiftPMManifestVersion;
+  } else {
+      return ctx.LangOpts.getMinPlatformVersion();
```

Comment:
More indents that should be 2 spaces. Also, can you add an assertion in the `else` branch?

Reply:
What would I assert on? The else case can be for both platform agnostic (unconditionally unavailable, deprecated etc) and platform specific availability.

Reply:
I think it's okay to assert that it's platform-specific and that it's never called for the unconditional kinds.